voltdbclient/authentication_test.go

```diff
@@ -23,21 +23,6 @@ import (
 	"testing"
 )
 
-func TestLoginRequest(t *testing.T) {
```

Comment:
@nshi my bad, the name of the test should have been `TestEncoder_Login` does this sound okay to you?

The answer to your question is no, this is the test for login encoded message